app/src/main/java/com/google/droidjump/MainActivity.java

```diff
@@ -18,12 +18,16 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.WindowManager;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        // Set full screen
+        getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
```

Comment:
What does this do exactly, we were not in full screen before?

Reply:
Without it we have a strange screen on a gameView, this line of code solves the problem.

Reply:
Could you attach screenshots for comparison?